calendar.css

```diff
@@ -14,9 +14,50 @@
     background-color: #4CAF50;
     color: white;
 }
-/* The following class has been added to reduce the height of the modal element. Otherwise, the modal element 
+/* The following class has been added to reduce the height of the modal element. Otherwise, the modal element
 	covers the entire table area which is not desirable. */
 .modal-height {
 	height: 240px;
 }
 
+.Xro3Db-nUpftc{position:absolute;top:0;right:0;bottom:0;left:0;overflow-y:auto;outline:none}
```

Comment:
please clean up this file by removing the extra classes, it's okay even if the ui looks bad because you can keep iterating on it and improving it. These css classes are auto-generated and obfuscated